mat2vec/training/phrase2vec.py

```diff
@@ -2,30 +2,46 @@
 from gensim.models.word2vec import LineSentence
 from gensim.models.phrases import Phrases, Phraser
 import gensim
-from mat2vec.training.helpers.utils import EpochSaver, compute_epoch_accuracies, \
-    keep_simple_formula, load_obj, COMMON_TERMS, EXCLUDE_PUNCT, INCLUDE_PHRASES
+from mat2vec.training.helpers.utils import EpochSaver, \
+    compute_epoch_accuracies, keep_simple_formula, \
+    load_obj, COMMON_TERMS, EXCLUDE_PUNCT, INCLUDE_PHRASES
 import logging
 import os
 import argparse
 import regex
 import pickle
 from tqdm import tqdm
 
-logging.basicConfig(format="%(asctime)s : %(levelname)s : %(message)s", level=logging.INFO)
+
+logging.basicConfig(
+    format="%(asctime)s : %(levelname)s : %(message)s",
+    level=logging.INFO)
 
 
 def exclude_words(phrasegrams, words):
-    """Given a list of words, excludes those from the keys of the phrase dictionary."""
+    """Given a list of words, excludes those from the keys of the phrase
+        dictionary.
+    """
     new_phrasergrams = {}
     words_re_list = []
     for word in words:
         we = regex.escape(word)
-        words_re_list.append("^" + we + "$|^" + we + "_|_" + we + "$|_" + we + "_")
-    word_reg = regex.compile(r""+"|".join(words_re_list))
+        words_re_list.append(
+            "^" +
+            we +
+            "$|^" +
+            we +
+            "_|_" +
+            we +
+            "$|_" +
+            we +
+            "_")
+    word_reg = regex.compile(r"" + "|".join(words_re_list))
     for gram in tqdm(phrasegrams):
         valid = True
         for sub_gram in gram:
-            if word_reg.search(sub_gram.decode("unicode_escape", "ignore")) is not None:
+            if word_reg.search(sub_gram.decode(
+                    "unicode_escape", "ignore")) is not None:
                 valid = False
                 break
```

Comment:
Thanks for paying attention to this. I'm also not sure what is happening here as I didn't wrote it. Since modifying this function might break something, I suggest to leave the logic the way it is. Also, we will probably use bio-related tokenizer in the future anyway.